devel/ci/integration/tests/fixtures/resultsdb.py

```diff
@@ -49,9 +49,7 @@ def resultsdb_container(docker_backend, docker_network, db_container, rabbitmq_c
     ]
     container = image.run_via_binary(additional_opts=run_opts)
     container.start()
-    # Add sample data in the database
-    container.execute(["resultsdb", "init_db"])
```

Comment:
This is what makes the container fail. The path to the `resultsdb` executable is different in this container though:
```patch
+    # Add sample data in the database
+    container.execute(["/venv/bin/resultsdb", "init_db"])
```